src/site/topic22.rst

```diff
@@ -26,6 +26,32 @@ Post-order
 Level-order
 -----------
 
+* A level order from the others
+
+* The idea is
+    * Start at the root
+    * Visit the nodes in each level from left to right
+
+* With this idea, there is no immediately obvious recursive definition of this traversal
+* An iterative definition of the traversal is perhaps simpler to derive
+
+* **Levelorder Traversal**
```

Comment:
Level order is written three different ways now (levelorder, level order, level-order). Pick one and go with it.